app/views/channels/_dimmer_hookup.html.erb

```diff
@@ -0,0 +1,45 @@
+<% @equipment = @equipment.sort_by &:dimmer %>
+<% equipment_groupings = @equipment.slice_when {|a, b| a.dimmer != b.dimmer}.to_a %>
+  <div id="dimmer-hookup" class="tab-pane fade">
+    <h3>Dimmer Hookup</h3>
+    <table class="table instrument-schedule">
+      <thead>
+        <tr>
+          <th class="Dim-H">Dim</th>
+          <th class="Chan-H">Channel</th>
+          <th class="Pos-H">Position</th>
+          <th class="Unit-H">Unit#</th>
+          <th class="Inst-H">Instrument Type</th>
+          <th class="Watt-H">Wattage</th>
+          <th class="Acc-H">Accessories</th>
+          <th class="Purp-H">Purpose</th>
+          <th class="Col-H">Color</th>
+          <th class="Ckt-H">Circuit</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% equipment_groupings.each do |equipment_group| %>
+          <tr><td colspan="10"><h4><%= equipment_group.first.dimmer %></h4></td></tr>
+          <div class="dim-groups">
+            <% equipment_group.each do |equip| %>
+              <tr id="DM-Row<%= equip.id %>">
+                <td class="e-pos"></td>
+                <td class="e-ch"><%= equip.channel %></td>
+                <td class="e-pos"><%= equip.position %></td>
+                <td class="e-num"><%= equip.unit_number %></td>
+                <td class="e-typ"><%= equip.instrument_type %></td>
+                <td class="e-wat"><%= equip.wattage %></td>
+                <td class="e-"><%= equip.accessories %></td>
+                <td class="e-pur"><%= equip.purpose %></td>
+                <td class="e-col"><%= equip.color %></td>
+                <td class="e-col"><%= equip.circuit_name %><%= equip.circuit_number %></td>
+                <td><a class="deep-blue edit-button" href="/productions/<%= equip.production_id %>/channels/<%= equip.id %>/edit"><span class="glyphicon glyphicon-edit" aria-hidden="true"></a></td>
+                <td><a class="deep-red delete-button" href="/productions/<%= equip.production_id %>/channels/<%= equip.id %>"><span class="glyphicon glyphicon-remove" aria-hidden="true"></a></td>
+              </tr>
+            <% end %>
+          </div> <!-- Close pos-groups -->
+        <% end %>
+      </tbody>
+    </table>
+  </div> <!-- Close dimmer hook-up -->
+
```

Comment:
consider a partial for the shared code between the circuit page and the dimmer page